testsuite/features/support/cobbler_test.rb

```diff
@@ -139,6 +222,13 @@ def exists(what, key, value)
     result
   end
 
+  ##
+  # It takes a string, a string, and a string, and returns a string
```

Comment:
This sounds really funny and should be adjusted :wink:

Reply:
I don't even feel good in suggesting anything here because I just love the laugh it gives me randomly looking at it :laughing: 
```suggestion
  # Retrieves an object from the server based on its type, name and key.
```